src/main/frontend/src/store/services/request.js

```diff
@@ -0,0 +1,37 @@
+import request from 'superagent';
+import { getTokenFromSessionStorage } from './getTokenFromSessionStorage';
+
+const get = (url, user = getTokenFromSessionStorage().token) => {
+  return request
+    .get(url)
+    .set('X-ID-TOKEN', user)
```

Comment:
```suggestion
    .set('X-ID-TOKEN', getTokenFromSessionStorage().token)
```

Double check what `getTokenFromSessionStorage()` returns.

Reply:
Same for other request types.